src/Cohort.js

```diff
@@ -0,0 +1,30 @@
+class Cohort {
+  constructor(cohortName, capacity = 24) {
+    this.cohortName = cohortName
+    this.studentInside = []
+    this.capacity = capacity
+  }
+
+  addStudent(student) {
+    this.studentInside.push(student)
+    return `${student.name} is now inside Cohort ${this.cohortName}`
+  }
+
+  removeStudent(student) {
+    const cohortIndex = this.studentInside.indexOf(student)
+    this.studentInside.splice(cohortIndex, 1)
```

Comment:
You'll want to check that the index exists before trying to splice it. Write a test for that, what happens if I try to remove a student that doesnt exist?